src/Page.tsx

```diff
@@ -10,10 +10,12 @@ interface Props {
 const Content = styled.section`
   grid-area: content;
   display: grid;
-  grid-template-areas:
-   "top-left top-right"
-   "bottom bottom";
+  grid-template-rows: 1fr 1fr auto;
```

Comment:
This extra 1fr is a remnant from the other PR, where I add a "global extra" (pets, equipment)